client/verta/docs/examples/tutorials/config.rst

```diff
@@ -0,0 +1,53 @@
+Client Config File
+==================
+The Client config file allows you to define settings that are easily repeatable and sharable. This
+can be either a `JSON <https://www.json.org/json-en.html>`__ or a `YAML <https://yaml.org/>`__ file.
+
+When the :class:`~verta.client.Client` is instantiated, it will look for a file called
+``verta_config.json`` or ``verta_config.yaml`` in the following locations—in order—using the
+first one it finds:
```

Comment:
This was supposed to merge them, so that you can keep things like devkey in your home but project name in the repository. Is that not how it's implemented?

Reply:
Ohhh I totally did not get that, the first few times I read your design doc for config. And the implementation _also_ doesn't merge them. I'll make a ticket!